packages/pg-vector-selector/tsconfig.json

```diff
@@ -26,6 +26,22 @@
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
     /* Modules */
     "module": "commonjs", /* Specify what module code is generated. */// "rootDir": "./",                                  /* Specify the root folder within your source files. */
+    "strict": true, /* Enable all strict type-checking options. */// "noImplicitAny": true,                            /* Enable error reporting for expressions and declarations with an implied 'any' type. */
+    "strictNullChecks": true,
```

Comment:
strict already includes strictNullCheck.
Is it just to make it more semantic?

Reply:
oh I don't know actually! but typia document recommend this

Reply:
Oh, I left out Nit

I thought it meant something

thank you for your answer

Reply:
Ah yeah this is Nit actually!
Every tsconfig.json in this project has this. I just formatted it!